src/_P020_Ser2Net.ino

```diff
@@ -288,6 +288,34 @@ boolean Plugin_020(uint8_t function, struct EventStruct *event, String& string)
         success = true;
       }
 
+      if (command == F("serialsendhex")){
+        success = true;
+        String tmpString = string.substring(14);
+        unsigned int len = tmpString.length();
+        const char* buf = tmpString.c_str();
+        for(unsigned int i=0; i < len / 2; i++){
+          uint8_t uc = buf[i * 2], dc = buf[i * 2 + 1];
+          if(uc >= 48 && uc <= 57) {
+              uc -= 48;
+          } else if(uc >= 65 && uc <= 70) {
```

Comment:
When using `parseString` the output is in lowercase already, so no need to check the UC cases